test/converter/excelHandlerSpec.js

```diff
@@ -189,10 +280,8 @@ describe('excelHandler', () => {
       expect(dmnContent.rules[0].outputEntries.length).to.eql(expectedRules[0].outputEntries.length);
       expect(dmnContent.rules[0].outputEntries[0].text).to.eql(expectedRules[0].outputEntries[0].text);
     });
-
   });
 
-
```

Comment:
Let's do not remove this line, as the tests always contain two spaces between two `describe` and `it`.

Reply:
[649c3e8](https://github.com/pinussilvestrus/camunda-modeler-excel-import-plugin/pull/109/commits/649c3e8730b4171d92714db46d9fb463e868b061)